letsencrypt/migrations/11.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, SUPERUSER_ID
+
+
+def migrate_altnames(env):
+    config = env["ir.config_parameter"]
+    existing = config.search([("key", "=like", "letsencrypt.altname.%")])
+    if not existing:
+        # We may be migrating from 10.0.2.0.0, in which case
+        # letsencrypt.altnames already exists and shouldn't be clobbered.
+        return
+    new_domains = "\n".join(existing.mapped("value"))
+    config.set_param("letsencrypt.altnames", new_domains)
+    existing.unlink()
+
+
+def migrate_cron(env):
+    # Any interval that was appropriate for the old version is inappropriate
+    # for the new one, so it's ok to clobber it.
+    # But tweaking it afterwards is fine, so noupdate="1" still makes sense.
+    jobs = (
+        env["ir.cron"]
+        .with_context(active_test=False)
+        .search([("model", "=", "letsencrypt"), ("function", "=", "cron")])
```

Comment:
This is giving errors on 11.0, as cron jobs are no longer directly using methods on models anymore, but using server actions:
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "bin/upgrade_odoo", line 63, in <module>
    sys.exit(anybox.recipe.odoo.runtime.upgrade.upgrade('/home/odorboprd11/odoo/upgrade.py', 'run', '/home/odorboprd11/odoo/etc/odoo.cfg', '/home/odorboprd11/odoo'))
  File "/home/odorboprd11/odoo/eggs/anybox.recipe.odoo-1.9.3.dev0-py3.7.egg/anybox/recipe/odoo/runtime/upgrade.py", line 151, in upgrade
    statuscode = getattr(upgrade_module, upgrade_callable)(session, logger)
  File "/home/odorboprd11/odoo/upgrade.py", line 19, in run
    session.update_modules(['all'])
  File "/home/odorboprd11/odoo/eggs/anybox.recipe.odoo-1.9.3.dev0-py3.7.egg/anybox/recipe/odoo/runtime/session.py", line 424, in update_modules
    self._registry = Registry(db)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/modules/registry.py", line 61, in __new__
    return cls.new(db_name)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/modules/registry.py", line 85, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/modules/loading.py", line 376, in load_modules
    force, status, report, loaded_modules, update_module, models_to_check)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/modules/loading.py", line 274, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/home/odorboprd11/odoo/parts/odoo/odoo/modules/loading.py", line 188, in load_module_graph
    migrations.migrate_module(package, 'post')
  File "/home/odorboprd11/odoo/parts/odoo/odoo/modules/migration.py", line 190, in migrate_module
    migrate(self.cr, installed_version)
  File "/home/odorboprd11/odoo/parts/server-tools/letsencrypt/migrations/11.0.2.0.0/post-migrate.py", line 38, in migrate
    migrate_cron(env)
  File "/home/odorboprd11/odoo/parts/server-tools/letsencrypt/migrations/11.0.2.0.0/post-migrate.py", line 25, in migrate_cron
    .search([("model", "=", "letsencrypt"), ("function", "=", "cron")])
  File "/home/odorboprd11/odoo/parts/odoo/odoo/models.py", line 1501, in search
    res = self._search(args, offset=offset, limit=limit, order=order, count=count)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/models.py", line 3838, in _search
    query = self._where_calc(args)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/models.py", line 3628, in _where_calc
    e = expression.expression(domain, self)
  File "/home/odorboprd11/odoo/parts/odoo/odoo/osv/expression.py", line 669, in __init__
    self.parse()
  File "/home/odorboprd11/odoo/parts/odoo/odoo/osv/expression.py", line 847, in parse
    raise ValueError("Invalid field %r in leaf %r" % (left, str(leaf)))
ValueError: Invalid field 'model' in leaf "<osv.ExtendedLeaf: ('model', '=', 'letsencrypt') on ir_cron (ctx: )>"
```

Reply:
Ah! I should have paid more attention there. It should be fixed now.